data/json/items/armor.json

```diff
@@ -5920,6 +5920,24 @@
     "material_thickness": 2,
     "flags": [ "VARSIZE", "WATER_FRIENDLY", "BELTED" ]
   },
+  {
+    "id": "leotard",
+    "type": "ARMOR",
+    "name": "leotard",
+    "name_plural": "leotard",
+    "description": "A skin-tight spandex garment, covering the body from the groin to the shoulders.   With its deep scoop-cut back and high waist cuts, this garment is perfect for showcasing your physique on the stage.",
```

Comment:
```suggestion
    "description": "A skin-tight spandex garment, covering the body from the groin to the shoulders.  With its deep scoop-cut back and high waist cuts, this garment is perfect for showcasing your physique on the stage.",
```

Reply:
I am surprised the JSON style checks didn't catch these. Thanks.